### DISCUSSION TITLE
Are There Restrictions on Participants Per User/Org?

### DISCUSSION THREAD

**Discussion by zicklag** [cyberbotics/wrestling#26] [Q&A] [answered]
Hey there, I found this a couple days ago and I think it's an amazing project!

I'm in the middle of bootstrapping what's necessary to create competitors in Rust, and I just pushed a [Rust wrestler template](https://github.com/katharostech/icra-2023-robot-wrestling-rust-template).

Before I created the robot for my team, I just wanted to make sure the system wasn't going to have problems with me publishing multiple robots from my GitHub organization ( at least one for the public template, and one private one for my team ). There aren't any rules against having multiple robots are there?

Thanks!

**Comment by omichel** [comment, marked as answer]
This is great that you are proposing such an example!
As added recently in the README.md file, there can be only one entry per individual. But we will make an exception for your public Rust example. So,  please go ahead and submit it. It is very welcome.
